cli/inspect/inspector_test.go

```diff
@@ -88,40 +156,93 @@ func TestInspect(t *testing.T) {
 		wantOut string
 		wantErr bool
 	}{
-		// TODO: Add test cases.
 		{
 			name: "testInspectDefault",
 			args: args{
-				references: "test",
+				references: []string{"single reference"},
 				tmplStr:    "",
 				getRef:     getRefFunc,
 			},
 			wantOut: "id",
 			wantErr: false,
 		}, {
-			name: "testInspectTemplate",
+			name: "testInspectTemplateSingleReference",
 			args: args{
-				references: "test",
+				references: []string{"single reference"},
 				tmplStr:    "{{.ID}}",
 				getRef:     getRefFunc,
 			},
 			wantOut: "id",
 			wantErr: false,
+		}, {
+			name: "testInspectTemplateSingleReferenceLiteralTemplate",
```

Comment:
IMO, the "testInspectTemplateSingleReferenceLiteral" should be enough

Reply:
Emmm, maybe you are right. I will reduce the test cases.